### ISSUE TITLE
Content inside BasePicker or BasePickerListBelow component is not visible in windows high contrast mode

### ISSUE THREAD

**Issue by KeiraYang** [microsoft/fluentui#15609]
### Environment Information

- **Package version(s)**: Latested version. Use fluent UI example page can repro
- **Browser and OS versions**: Edge Chromium, Windows 10

### Describe the issue:

Content inside BasePicker or BasePickerListBelow component is not visible in windows high contrast mode.
E.g. When Persona component is used to render suggested items.

### Please provide a reproduction of the issue in a codepen:

1. Go to Windows High contrast settings -> Turn on High contrast -> Choose theme "High Contrast White".
2. Go to https://codepen.io/keirayang/pen/YzWGwzz.
3. Typing "d" in the input box.
4. Observe the issue.

#### Actual behavior:

![image](https://user-images.githubusercontent.com/28148759/96563002-03342000-12f4-11eb-90d3-48a649f88802.png)

#### Expected behavior:

It should works like people picker example or tag picker example:
https://developer.microsoft.com/en-us/fluentui#/controls/web/peoplepicker
![image](https://user-images.githubusercontent.com/28148759/96563087-1f37c180-12f4-11eb-8221-5eae0d49347d.png)

**Comment by KeiraYang** [comment]
@tomi-msft Could you please help take a look and confirm if this is BasePicker bug? Is there any plan to fix the issue? Thank you

**Comment by paulgildea** [comment]
@KeiraYang Thanks for filing this issue!

@xugao @smhigley FYI

**Comment by tomi-msft** [comment]
Hi @KeiraYang thanks for filing the issue. This is not a BasePicker bug, but rather an issue with the Picker example. The text is not styled properly to support High Contrast. I will fix the example in our website. For your own case you should provide your own styles to the custom picker(similar to how PeoplePicker is styled) and that should fix any styling issues you have in hc mode.
